### ISSUE TITLE
New feature - allow-reset-consumergroup-for-all-topics

### ISSUE THREAD

**Issue by sunshine69** [deviceinsight/kafkactl#127]
Hi,

I would like to reset a offset applied to all topics - my initial proposal is for all, but I would like to limit it in a list of topics as well , please comment, if it is ok then I would create next commit to support the list.

Basically it enables you to do something like this

```
./kafkactl reset consumer-group-offset test-kafka --newest --all-topics
```

**Comment by sunshine69** [comment]
For initial PR https://github.com/deviceinsight/kafkactl/pull/128 - if the implementation direction is okay I would complete the list (changelog etc ...)

Thank you.

**Comment by d-rk** [comment]
Hi @sunshine69,
thank you for your PR. The approach looks good, so we would be happy to see it being completed.

**Comment by d-rk** [comment]
implemented with [v2.3.0](https://github.com/deviceinsight/kafkactl/releases/tag/v2.3.0)
